documentation/Background.md

```diff
@@ -0,0 +1,78 @@
+# 1) Background Info
+### 1.1) What is Trust?
+Trust is when the computer does something on our behalf we have to trust it with our data. A trusted component is that which always behaves in an expected manner.  To make a component "trusted", you must establish some root of trust. A Root of Trust Measurement is the process of measuring (hashing) components on a system to validate their integrity. When establishing a root of trust, there are two primary schools of thought: Static Root of Trust Measurements and Dynamic Root of Trust Measurements.
```

Comment:
@garnetgrimm it would be way easier to review if the document would be wrapped at 80 characters